CHANGELOG.md

```diff
@@ -7,12 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Add migrate script to help migrate to the new Ray Trainer, run "python -m deepgnn.migrate.ray_torch --main_path <your main.py with run_dist>" to convert it to use the Ray Trainer. Add --hvd to convert to horovod trainer.
```

Comment:
Please, keep only versions in the migrate script. It should be just `python -m deepgnn.migrate.0.1.58`. If you want to do ray explicitly, use command line flags instead.